sky/volumes/utils.py

```diff
@@ -107,7 +107,9 @@ def _add_rows(self,
                 size,
                 row.get('user_name', '-'),
                 row.get('workspace', '-'),
-                log_utils.readable_time_duration(row.get('launched_at', 0)),
+                log_utils.readable_time_duration(start=row.get(
+                    'launched_at', 0),
+                                                 absolute=True),
```

Comment:
This still shows a very long string (e.g., `4h 10m 46s`). Can we follow kubectl convention which removes insignificant bits? 

<img width="69" alt="image" src="https://github.com/user-attachments/assets/e9c11ac1-0f8c-407f-92b4-0003baf51c8e" />